src/sentry/static/sentry/app/utils/performance/quickTrace/types.ts

```diff
@@ -38,20 +37,31 @@ export type QuickTraceEvent = EventLite & {
 /**
  * The `events-trace` endpoint returns a tree structure that gives
  * the parent-child relationships between events.
+ *
+ * This is the type returned with `detailed=0`
  */
-export type TraceFull = Omit<QuickTraceEvent, 'generation' | 'errors'> & {
+export type TraceFullMinimal = Omit<QuickTraceEvent, 'generation' | 'errors'> & {
```

Comment:
nit (doesn't matter if you don't want to change this): `TraceFullMinimal` is a bit confusing, can we change `TraceFull` to maybe something like `TraceTree`? Or something else so that we don't have a bit of an oxymoron in the name.